test/test_torch.py

```diff
@@ -1353,7 +1353,7 @@ def test_inference(default_dtype):
             self.assertIs(default_dtype, torch.tensor(((7, 5), (9, 5.))).dtype)
             self.assertIs(default_dtype, torch.tensor(((5., 5), (3, 5))).dtype)
             self.assertIs(torch.int64, torch.tensor(((5, 3), (3, 5))).dtype)
-            self.assertIs(torch.complex128, torch.tensor(((5, 3 + 2j), (3, 5 + 4j))).dtype)
+            self.assertIs(torch.complex64, torch.tensor(((5, 3 + 2j), (3, 5 + 4j))).dtype)
```

Comment:
This looks wrong, it should be based on `default_dtype`; `torch.complex128` if it's double default dtype, and 64 otherwise

Reply:
updated!